app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V1::ReviewsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def create
+    data = JSON.parse(request.body.read)
+    new_review = Review.create(
+      body: data["body"],
+      rating: data["rating"],
+      location_id: params[:location_id],
+      user_id: current_user.id,
```

Comment:
Again - this should have some logic to check for `current_user` and render json errors if things don't work out